gravitee-apim-console-webui/src/management/home/home-overview/home-overview.component.ts

```diff
@@ -207,27 +205,16 @@ export class HomeOverviewComponent implements OnInit, OnDestroy {
         },
       });
 
-    // Request Stats
+    // Request Stats v4
```

Comment:
```suggestion
    // Request Stats
```

it is not only v4 🤔 ?